src/pohjavirta/response.clj

```diff
@@ -97,6 +99,28 @@
         (.getResponseSender)
         (.send ^ByteBuffer body)))
 
+  InputStream
+  (send-body [stream exchange]
+    (let [bytes (byte-array 8192)
+          send-more (fn send-more [^Sender sender]
+                      (let [count (.read stream bytes 0 8192)]
+                        (cond
+                          (< 0 count)
+                          (.send sender
+                                 (ByteBuffer/wrap bytes 0 count)
+                                 (reify IoCallback
+                                   (onComplete [_ _ sender]
+                                     (send-more sender))
```

Comment:
Does this add a new stack frame for each 8kb? If so, it will eventually blow up. In older versions of UnderTow, you could say `.startBlocking` on exchange (should happen do this in the worker pool) after which you can call `.getOutputStream` to write the Stream to. Some pointers:

- https://github.com/ikitommi/immutant/blob/master/web/src/immutant/web/internal/undertow.clj#L175-L178
- https://github.com/ikitommi/immutant/blob/master/web/src/immutant/web/internal/ring.clj#L143-L155